.travis.yml

```diff
@@ -1,37 +1,46 @@
 sudo: required
 dist: trusty
 
+env:
+    global:
+        - ENCRYPTION_LABEL: "79272d66a36b"
+        - COMMIT_AUTHOR_EMAIL: "travis@coala.io"
+
 language: python
 node_js:
-  - "4.1"
+    - "4.1"
```

Comment:
Hey this is not needed, it just makes some noise on the diff.

Reply:
Ooh, I just ran YAMLLintBear and applied patch to it without checking anything. Will fix everything.